packages/commonwealth/package.json

```diff
@@ -212,8 +212,10 @@
     "stream-http": "^3.2.0",
     "superagent": "^5.2.2",
     "swagger-ui-express": "^5.0.0",
-    "tsx": "^4.7.2",
+    "ts-node": "^10.9.2",
```

Comment:
can't we use tsx instead? seems like tsx and heroku are not friends? should be build and use node index.js? ... just looking for the best pattern forward

Reply:
`ts-node-dev` as well. Otherwise this will break Rays upcoming work.